### PR TITLE
WIP: Added tests for stack analyses endpoint

### PR DESCRIPTION
**Pull request by pkajaba**

### CODE REVIEW

File: integration-tests/feature_list.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #features/packages.feature
 #features/versions.feature
 #features/anitya.feature
+#features/stack-analyses.feature
```
Comment (jpopelka) [inline integration-tests/feature_list.txt]: You haven't added the `stack-analyses.feature` file itself.

File: integration-tests/features/steps/common.py [hunk 2]
```diff
@@ -213,44 +201,48 @@ def validate_analysis_result(context, ecosystem, package, version):
         jsonschema.validate(struct, schema)
 
 
+@when("I get a valid request ID")
 @then("I should get a valid request ID")
```
Comment (jpopelka) [inline integration-tests/features/steps/common.py]: Is it OK that there's `@when` and `@then` at the same time ?

File: integration-tests/features/steps/common.py [hunk 3]
```diff
@@ -213,44 +201,48 @@ def validate_analysis_result(context, ecosystem, package, version):
...
-    jsonschema.validate(resp_json, schema)
+    assert context.response['status'] == "success"
+    assert len(context.response['id']) == 32
+
+
+@when("I post a valid {manifest} to {url}")
+def perform_valid_manifest_post(context, manifest, url):
+    files = {'manifest[]': open("data/poms/{manifest}".format(manifest=manifest.replace("\"", '')), 'r')}
+    import os
+    print(os.curdir)
```
Comment (jpopelka) [inline integration-tests/features/steps/common.py]: left-over ?

File: integration-tests/features/steps/common.py [hunk 4]
```diff
@@ -213,44 +201,48 @@ def validate_analysis_result(context, ecosystem, package, version):
...
+        files=files
+    ).json()
+    context.response.raise_for_status()
+
+
+@then("I get stack analyses response via {url}")
+def get_stack_analyses_response(context, url):
+    url = "{base_url}{url}{request_id}".format(
+        base_url=context.coreapi_url,
+        url=url, request_id=context.response['id'])
```
Comment (jpopelka) [inline integration-tests/features/steps/common.py]: `context.response.json()['id']` perhaps ?

File: integration-tests/feature_list.txt (hunk 1 above)
Comment (pkajaba) [inline integration-tests/feature_list.txt]: ups, it should be there now

File: integration-tests/features/steps/common.py (hunk 2 above)
Comment (pkajaba) [inline integration-tests/features/steps/common.py]: I saw similar usage here: https://github.com/pkajaba/fabric8-analytics-common/blob/1e628e3f61beccf4510385a1f4b6722f89002891/integration-tests/features/steps/common.py#L155 so I assumed it's OK. I have to check it out

File: integration-tests/features/steps/common.py (hunk 3 above)
Comment (pkajaba) [inline integration-tests/features/steps/common.py]: damn, yes, I am sorry

File: integration-tests/features/steps/common.py (hunk 4 above)
Comment (pkajaba) [inline integration-tests/features/steps/common.py]: I jsonify response when I store it. No need to do this

**Pull request closed without merging**